File: TaskMasterWeb/TaskMasterWeb.Client/Dtos/Project/ProjectTaskDto.cs
namespace TaskMasterWeb.Client.Dtos.Project;

public record ProjectTaskDto
{
    public int Id { get; set; }

    public string Name { get; set; } = default!;

    public string Status { get; set; } = default!;

    public string Priority { get; set; } = default!;

    public string Assignee { get; set; } = default!;
}

File: TaskMasterAPI/src/main/java/com/devlab/taskmaster/dto/project/ProjectTaskDTO.java
package com.devlab.taskmaster.dto.project;

import com.devlab.taskmaster.enumeration.TaskPriority;
import com.devlab.taskmaster.enumeration.TaskStatus;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class ProjectTaskDTO {

    private Integer id;

    private String name;

    private TaskStatus status;

    private TaskPriority priority;

    private String assignee;
}
